commands/action.go

```diff
@@ -480,6 +480,9 @@ func parseAction(cmd *cobra.Command, args []string, update bool) (*whisk.Action,
 		return nil, noArtifactError()
 	}
 
+	if update {
+		action.DelAnnotations = Flags.action.delAnnotation
```

Comment:
I am not clear about this, can users pass list of annotations? or they have to use `--del-annotation` for every annotation?

Reply:
Must pass like this `--del-annotation key1 --del-annotation key2`, in go backend, it will recevie the value by array:
https://github.com/apache/openwhisk-client-go/pull/137/files#diff-7a2fb893e2fa76731b906c5689a14e95R39